src/Token/_tokenButtonUtils.ts

```diff
@@ -0,0 +1,61 @@
+import {css} from 'styled-components'
+import {variant} from 'styled-system'
+import {get} from '../constants'
+import {tokenSizes, TokenSizeKeys, defaultTokenSize} from './TokenBase'
+
+export interface TokenButtonProps {
+  borderOffset?: number
+  parentTokenTag: 'span' | 'button' | 'a'
+  variant?: TokenSizeKeys
+  parentTokenIsInteractive?: boolean
```

Comment:
Using `is` as a prefix for booleans feels more consistent here:

```suggestion
  isParentInteractive?: boolean
```

I'm also unsure if having "token" in the variable name is redundant or not.